schemas/plugins/debug/ProfilingPlugin.json

```diff
@@ -6,8 +6,7 @@
     "outputPath": {
       "description": "Path to the output file e.g. `profiling/events.json`. Defaults to `events.json`.",
```

Comment:
Examples should showcase an absolute path. `path.resolve(__dirname, "profiling/events.json)`